src/driver/verify.cpp

```diff
@@ -86,6 +88,7 @@ std::vector<argument> run_target(program p,
                                  precision quantize,
                                  const parameter_map& inputs)
 {
+    // NOTE: why do we have this here when driver/main.cpp:compiler already quantized
```

Comment:
It doesn't quantize it for `verify`:

https://github.com/ROCm/AMDMIGraphX/blob/develop/src/driver/main.cpp#L579

So the ref is not quantized whereas the target is quantized.